pageserver/src/tenant.rs

```diff
@@ -2086,6 +2087,10 @@ impl Tenant {
         &self.tenant_shard_id
     }
 
+    pub(crate) fn get_shard_stripe_size(&self) -> ShardStripeSize {
+        self.shard_identity.stripe_size
+    }
```

Comment:
nit: any reason why you don't expose `shard_identity` and let the caller peel out the strip size? AFAICT it's an immutable attribute

Reply:
I'm sort of ambivalent about public-immutable members vs getters.  In this instance I leaned toward a getter because it was just one code path that needed it, so if we ever refactor that we'll get a nice clippy notification that  we can revert this to be totally private.